codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -317,8 +317,10 @@ update these file paths accordingly.
         ? fileSystem.path.join(parent.path, 'entitlements.txt')
         : fileSystem.path.join(parent.path, 'without_entitlements.txt');
     if (!(await fileSystem.file(entitlementFilePath).exists())) {
-      throw CodesignException('$entitlementFilePath not found \n'
-          'make sure you have provided them along with the engine artifacts \n');
+      log.warning('$entitlementFilePath not found. '
```

Comment:
The result is that no binaries will be signed, right? I think we should make that clear in this warning message (it was not clear to me until I read the PR description).

Reply:
Good advice! Added a line informing user no binaries will be signed.